Core/Core/Database.xcdatamodeld/Database.xcdatamodel/contents

```diff
@@ -176,6 +176,58 @@
         <relationship name="course" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="Course" inverseName="contextColor" inverseEntity="Course"/>
         <relationship name="groups" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="Group" inverseName="contextColor" inverseEntity="Group"/>
     </entity>
+    <entity name="ContextEnrollment" representedClassName="Core.ContextEnrollment" syncable="YES">
```

Comment:
These entities are specific to the people list screen so I'd prefix them with that: PeopleListEnrollment, PeopleListUser. This way it will be straightforward what these are good for.

Reply:
done.